addons/packages/packetfence.spec

```diff
@@ -119,6 +119,8 @@ Requires: perl(Data::Phrasebook), perl(Data::Phrasebook::Loader::YAML)
 Requires: perl(DBI)
 Requires: perl(Rose::DB)
 Requires: perl(Rose::DB::Object)
+Requires: perl(Lingua::EN::Nums2Words) >= 1.16
```

Comment:
Are these packages built and in the packetfence-devel repo?

Reply:
They are built not but not in packetfence-devel